katrain/core/engine.py

```diff
@@ -311,7 +320,9 @@ def request_analysis(
         next_move: Optional[GameNode] = None,
         extra_settings: Optional[Dict] = None,
         report_every: Optional[float] = None,
+        current_analysis: bool = False,
     ):
+        self.analysis_node = analysis_node
```

Comment:
remove, the engine may be working on any number of nodes and the last query is not necessarily the currnet node